docs/dev/setup/localci-localvc.rst

```diff
@@ -143,11 +143,78 @@ Setup with Docker Compose
+
+`Podman <https://podman.io/>`_ offers a container runtime that is API-compatible with Docker.
+Rather than having a system-wide socket that runs with administrative permissions, though, Podman allows to create containers with only user permissions.
+In single-user setups this might not be as relevant, but offers additional security in a production environment where the Artemis CI has to execute untrusted student code.
+
+.. admonition:: Podman is supported on a best effort basis.
+
+    We are relying on the API compatibility to provide support but are not actively testing against Podman on a test system or in the continuous integration.
+    If you notice any issues, feel free to open an issue or pull request so that we can try to fix them.
+
+.. note::
+
+    These setup steps are mostly focused on Linux systems.
+    On mac and Windows, both Docker and Podman run the containers in a small virtual machine anyway.
+    Therefore, there is little technical benefit relevant to Artemis for choosing one over the other in local development setups.
+    If in doubt, we recommend to use Docker, since that solution is most likely to be tested by other Artemis developers.
+
+
+Linux setup
+"""""""""""
+
+Podman itself should be available via your regular package manager.
+
+After installation, you have to ensure that your user is allowed to create containers.
+This is managed by the files ``/etc/subuid`` and ``/etc/subgid``.
+Ensure both files contain a line starting with your username.
+If not, you can generate the relevant lines using
+
+.. code-block:: bash
+
+    #! /usr/bin/env sh
+
+    printf "%s:%d:65536\n" "$USER" "$(( $(id -u) * 65536 ))" | tee -a /etc/subuid /etc/subgid
+
+After that, enable the Podman user socket that provides the API for the container management:
+
+.. code-block:: bash
+
+    systemctl --user enable --now podman.socket
+
+Configure the connection to this socket in Artemis by replacing ``${UID}`` with your actual user id (``id -u``):
+
+.. code-block:: yaml
+
+    artemis:
+        continuous-integration:
+            docker-connection-uri: "unix:///run/user/${UID}/podman/podman.sock"
+            # alternatively, if you use the `DOCKER_HOST` environment variable already
+            # to tell other tools to use the Podman socket instead of the Docker one:
+            # docker-connection-uri: "${DOCKER_HOST}"
+
+
+Windows or mac setup
+""""""""""""""""""""
+
+Podman offers a `desktop application <https://podman-desktop.io/>`_ application similar to Docker desktop and `CLI tools <https://podman.io>`_ for Windows, macOS, and Linux.
+As with Docker, to run containers on Windows or macOS the runtime has to start a small virtual Linux machine that then actually runs the containers.
+You can probably connect to this VM similarly as described in the regular setup steps above
+(`additional Podman documentation <https://podman-desktop.io/docs/migrating-from-docker/using-the-docker_host-environment-variable>`_).
+
+.. note::
```

Comment:
Not sure if this fits here, but it feels a bit out of place ot me. Maybe move to a preface at the start of the whole section? Maybe others have better ideas!